crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -364,25 +371,37 @@ impl Iterator for LiveSetIter<'_> {
 
 // These functions are used to initialize the DB tables
 fn owned_object_transaction_locks_table_default_config() -> DBOptions {
-    point_lookup_db_options()
+    optimized_for_high_throughput_options(
+        read_size_from_env(ENV_VAR_LOCKS_BLOCK_CACHE_SIZE).unwrap_or(1024),
+        false,
+    )
 }
 
 fn objects_table_default_config() -> DBOptions {
-    let db_options = point_lookup_db_options();
     DBOptions {
-        options: db_options.options,
+        options: optimized_for_high_throughput_options(
+            read_size_from_env(ENV_VAR_OBJECTS_BLOCK_CACHE_SIZE).unwrap_or(5 * 1024),
```

Comment:
should we also cover the `indirect_move_objects` table?

Reply:
sure, I'll add it.